apps/web/src/types/Pool.ts

```diff
@@ -0,0 +1,6 @@
+export type Pool =
```

Comment:
Let's export as an interface and move it to that directory. Also change Pool.pool to Pool.address.